tests/Doctrine/ODM/MongoDB/Tests/Aggregation/Stage/LookupTest.php

```diff
@@ -420,4 +420,50 @@ private function insertTestData()
         $this->dm->persist($referenceUser);
         $this->dm->flush();
     }
+
+    public function testLookupStageAndDefaultAlias()
+    {
+        $builder = $this->dm->createAggregationBuilder(\Documents\User::class);
+        $builder
+            ->lookup('simpleReferenceOneInverse');
+
+        $expectedPipeline = [
+            [
+                '$lookup' => [
+                    'from' => 'SimpleReferenceUser',
+                    'localField' => '_id',
+                    'foreignField' => 'userId',
+                    'as' => 'simpleReferenceOneInverse',
+                ],
+            ],
+        ];
+
+        $this->assertEquals($expectedPipeline, $builder->getPipeline());
+
+        $result = $builder->execute()->toArray();
+        $this->assertCount(1, $result[0]['simpleReferenceOneInverse']);
+    }
+
+    public function testLookupStageAndDefaultAliasOverride()
+    {
+        $builder = $this->dm->createAggregationBuilder(\Documents\User::class);
+        $builder
+            ->lookup('simpleReferenceOneInverse')->alias('override');
```

Comment:
Please fix indentation:
```suggestion
            ->lookup('simpleReferenceOneInverse')
                ->alias('override');
```